packages/card/style.module.css

```diff
@@ -13,11 +13,15 @@
   position: relative;
   background-color: var(--background-color);
   transition: 0.2s ease;
-  transition-property: border-color, background-color;
+  transition-property: transform, border-color, box-shadow, background-color;
+  box-shadow: 0 16px 24px -16px #cbcbcd33, 0 8px 12px -8px #cbcbcd66;
 
   &:hover {
     --border-color: var(--wpl-neutral-300);
 
+    box-shadow: 0 16px 16px -10px #656a7633, 0 2px 3px 0 #656a7626;
+    transform: translateY(-2px);
```

Comment:
Doom flicker https://www.joshwcomeau.com/animation/css-transitions/#doom-flicker


https://user-images.githubusercontent.com/825855/198085740-20c7c333-5aeb-48e6-9211-dbd295cc626b.mp4

Reply:
😱 Good catch! Thanks for that link as well, it was useful in understanding the issue in greater detail :+1: 

Addressed here:
https://github.com/hashicorp/react-components/pull/776/commits/6b507ce9f1784b2c0acfca9c142894af289a336c